tools/run-clang-format.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env python
```

Comment:
Heads up, you need to paste their license somewhere. Checkout https://github.com/SwarmUS/HiveMind/blob/feat/logging/THIRD-PARTY-NOTICES.md

Reply:
Added esp-idf license as well